### ISSUE TITLE
`chdir` on `docker_compose_exec` missing pass-through

### ISSUE THREAD

**Issue by smoberly** [ansible-collections/community.docker#517]
The `chdir` parameter seems to have been missed during the rewrite. The following line needs to be added:

```
--- a/plugins/modules/docker_container_exec.py
+++ b/plugins/modules/docker_container_exec.py
@@ -249,6 +249,7 @@ def main():
             'AttachStderr': True,
             'Cmd': argv,
             'Env': format_environment(env) if env is not None else None,
+            'WorkingDir', chdir or '',
         }
         exec_data = client.post_json_to_json('/containers/{0}/exec', container, data=data)
         exec_id = exec_data['Id']
```

**Comment by felixfontein** [comment]
Thanks for reporting this! I've created a fix in #518. The fix needs to be a bit more complicated though since the module also supports API < 1.35, which does not accept the `WorkingDir` argument. (I'm not sure whether the API would actually complain if it is still specified, but better safe than sorry... :) )

**Comment by smoberly** [comment]
Yeah, the previous code base had that exception in there too. So, I just pointed to the problem.  Thanks for the quick response.
